site/src/theme/Root.js

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from 'react';
+
+const MESSAGE_TYPE = 'estuary.docs.hideNavBar';
+
+const BODY_CLASS = 'running-in-iframe';
+
+// Default implementation, that you can customize
+export default function Root({ children }) {
```

Comment:
This is in a stand alone file as `root` so we have a simple place to put all the "onLoad" stuff we need for our Docs